querydsl-spatial/src/main/java/com/querydsl/spatial/path/GeometryPath.java

```diff
@@ -139,7 +139,7 @@ public GeometryPath(String var) {
         return polygon;
     }
 
-    public PolyhedralSurfacePath<PolyHedralSurface> asPolygHedralSurface() {
+    public PolyhedralSurfacePath<PolyHedralSurface> asPolyHedralSurface() {
```

Comment:
If you were to fix this, I'd do it in a separate commit if I were you.
Otherwise you can't easily backport it.